api/models/api.articles.models.js

```diff
@@ -81,19 +80,51 @@ exports.fetchCommentsByArticleId = (articleId) => {
       [articleId]
     )
     .then((comments) => {
-      const promiseAll = [comments.rows]
-      if(comments.rows.length === 0){
-        const checkArticleId = db.query('SELECT * FROM articles WHERE article_id = $1',[articleId])
-        promiseAll.push(checkArticleId)
+      const promiseAll = [comments.rows];
+      if (comments.rows.length === 0) {
+        const checkArticleId = db.query(
+          "SELECT * FROM articles WHERE article_id = $1",
+          [articleId]
+        );
+        promiseAll.push(checkArticleId);
       }
-      return Promise.all(promiseAll)
-    }).then(([comments, checkArticleId])=>{
-      if(comments.length > 0)return comments
+      return Promise.all(promiseAll);
+    })
+    .then(([comments, checkArticleId]) => {
+      if (comments.length > 0) return comments;
 
-      if(checkArticleId.rows.length > 0){
-        return []
-      }else{
-        return Promise.reject({ status:404,msg:'Not Found'})
+      if (checkArticleId.rows.length > 0) {
+        return [];
+      } else {
+        return Promise.reject({ status: 404, msg: "Not Found" });
       }
+    });
+};
+
+exports.addCommentByArticleId = (articleId, body) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id =$1", [articleId])
+    .then((checkArticleIdExists) => {
```

Comment:
this is a slightly misleading variable name - generally when we use a verb in javascript like "check" it is for a function. The variable here is the article or articles (ie. use nouns for const/let variables)